examples/seedpython/scripts/string.py

```diff
@@ -0,0 +1,31 @@
+"spam eggs"  # spam eggs
```

Comment:
For strings, I think we need to clarify if the following string literals are supported or not in the comments of this file:

* raw string like r"..."
* unicode string like u"..."
* string templates like "...%()...", "...{}..." or f"...{}..."

Reply:
Added comments to state all these syntaxes are not supported now.